packages/cli/src/commands/doctor/healthchecks/iosDeploy.ts

```diff
@@ -1,58 +1,116 @@
 import execa from 'execa';
+import chalk from 'chalk';
+// @ts-ignore untyped
+import inquirer from 'inquirer';
 import {checkSoftwareInstalled, PACKAGE_MANAGERS} from '../checkInstallation';
 import {packageManager} from './packageManagers';
-import {logManualInstallation} from './common';
+import {logManualInstallation, removeMessage} from './common';
 import {HealthCheckInterface} from '../types';
+import ora from 'ora';
 
-const getInstallationCommand = () => {
+const label = 'ios-deploy';
+
+const installationWithYarn = 'yarn global add ios-deploy';
+const installationWithNpm = 'npm install ios-deploy --global';
+
+const identifyInstallationCommand = () => {
   if (packageManager === PACKAGE_MANAGERS.YARN) {
-    return 'yarn global add ios-deploy';
+    return installationWithYarn;
   }
 
   if (packageManager === PACKAGE_MANAGERS.NPM) {
-    return 'npm install ios-deploy --global';
+    return installationWithNpm;
   }
 
   return undefined;
 };
 
+const installLibrary = async ({
+  installationCommand,
+  packageManagerToUse,
+  loader,
+}: {
+  installationCommand: string;
+  packageManagerToUse: 'yarn' | 'npm';
+  loader: ora.Ora;
```

Comment:
You can use this if you prefer (that's what I did for the rest of the command)

```typescript
import {Ora} from 'ora';
```

```typescript
loader: Ora;
```

Reply:
Done 🙂